xdn-sapper-example/webpack.config.js

```diff
@@ -0,0 +1,97 @@
+const webpack = require('webpack');
```

Comment:
Why have a rollup and webpack config?

Reply:
Rollup was left as a backup example, but it is now removed.